docs/tutorial/tasks.md

```diff
@@ -210,12 +200,9 @@ Average open issues per user 💌 : 2.27
 
 </div>
 
-## Subflows
-
-Not only can you call tasks within a flow, but you can also call other flows!
-Child flows are called [subflows](/concepts/flows/#composing-flows) and allow you to efficiently manage, track, and version common multi-task logic.
+## Organize multiple tasks
```

Comment:
```suggestion
## Organize multiple flows
```

I think the intent of this section is more about organizing multiple flows rather than multiple tasks, but subflows could be used to organize a bunch of tasks.